content/03-types/02-relational/07-what-are-joins.mdx

```diff
@@ -0,0 +1,401 @@
+| 1           | Jenny   | 30  | jenny@prisma.io   | 1        | $150.00 | 4/1/2024   |
+| 2           | Adrian  | 28  | adrian@prisma.io  | 2        | $200.00 | 3/29/2024  |
+| 1           | Jenny   | 30  | jenny@prisma.io   | 3        | $20.00  | 4/2/2024   |
+| 3           | Synniva | 25  | synniva@prisma.io | 4        | $320.00 | 4/4/2024   |
+
+The result set combines the columns from both the `orders` table and `customers` table based on the matching rows from the `customer_id` column. This allows you to see order information alongside the customer information.
+
+## Different Types of JOINs in MySQL and PostgreSQL
+
+Sometimes you may want to get different rows from each table. There are different types of JOIN operations that will help you achieve this. In this section, we will discuss traditional JOINs supported by MySQL and PostgreSQL (two relational databases) as well as lateral JOINs (only supported by PostgreSQL).
+
+### Traditional JOINs
+
+The most common form of JOIN operations you will encounter are `INNER JOIN`, `LEFT JOIN`, `RIGHT JOIN`, `FULL JOIN`, and `CROSS JOIN`. To demonstrate the following types of JOINs, we will use the following `customers` and `orders` (modified) tables:
+
+| customer_id | name    | age | email address     |
+| ----------- | ------- | --- | ----------------- |
+| 1           | Jenny   | 30  | jenny@prisma.io   |
+| 2           | Adrian  | 28  | adrian@prisma.io  |
+| 3           | Synniva | 25  | synniva@prisma.io |
+
+| order_id(primary key) | cost    | order_date | customer_id(foreign key) |
+| --------------------- | ------- | ---------- | ------------------------ |
+| 1                     | $150.00 | 4/1/2024   | 1                        |
+| 2                     | $200.00 | 3/29/2024  | 2                        |
+| 3                     | $20.00  | 4/2/2024   | 1                        |
+
+#### **INNER JOIN**
+
+The `INNER JOIN` is the default JOIN and returns rows from both tables only where there is a match. Here is the associated query for the `customers` and `orders` table:
+
+```sql
+SELECT
+    *
+FROM
+    customers
+[INNER] JOIN orders
+    ON customers.customer_id = orders.customer_id;
+```
+
+Here is the result table after using an `INNER JOIN`:
+
+| customer_id | name   | age | email address    | order_id | cost    | order_date |
+| ----------- | ------ | --- | ---------------- | -------- | ------- | ---------- |
+| 1           | Jenny  | 30  | jenny@prisma.io  | 1        | $150.00 | 4/1/2024   |
+| 1           | Jenny  | 30  | jenny@prisma.io  | 3        | $20.00  | 4/2/2024   |
+| 2           | Adrian | 28  | adrian@prisma.io | 2        | $200.00 | 3/29/2024  |
+
+When depicted as a Venn diagram, an `INNER JOIN` represents the overlapping region of the two circles. In other words, only the values existing in both tables are included.
+
+![inner-join](../../dataguide-images/types/relational/what-are-joins/inner-join.png)
```

Comment:
for this image and following (left, right, full join), can we maybe make them a bit smaller? They feel disproportionately big compared to the rest. Maybe a 50-70% size?